### ISSUE TITLE
Characters deleted in-game still show up in TQVaultAE causing error

### ISSUE THREAD

**Issue by Pax1sw** [EtienneLamoureux/TQVaultAE#130]
The reason I got TQVaultAE was so I could delete all my "Holder" characters, but now, after deleting some of them, I run TQVaultAE and it still lists those deleted characters as being there. Of course, they're not, so TQVaultAE returns multiple duplicate error messages for those deleted characters, which are nonetheless STILL listed in the drop-down after TQVaultAE boots. There's no "Reload Characters" option under Configuration, and unchecking "Pre-load" then exiting doesn't initiate a refresh when it's checked again, with yet another exit and reboot of TQVaultAE. Could you include a "Refresh/Reload Character List" in Configuration?

**Comment by Pax1sw** [comment]
Not sure you need it, but this is a screen capture of the multiple errors.
![Character Load Error](https://user-images.githubusercontent.com/50997413/58585931-8cd6df80-821f-11e9-94b1-86157d3598ff.jpg)

**Comment by EtienneLamoureux** [comment]
Thanks for the screen captures. You could always manually delete the characters folders causing problem by going to the path listed at the top of the error.

**Comment by Pax1sw** [comment]
OK! I manually removed from that directory all the characters I deleted in-game and am no longer getting those error messages on start-up. YAY! I think TQVaultAE was running when I deleted them in-game, did not know it should not have been. Was a real chore clearing their inventories and transferring it all to TQVaultAE, since I had to repeat the process something like 24 times (according to the error messages I counted at TQVaultAE start-up), so kept TQVaultAE open.

Strangely, the characters I deleted from the game that did not have "Holder" as their 1st name seem to have been removed from the folder properly, since they were no longer there when I checked. Perhaps I didn't have TQVaultAE open during their deletions.

I'll make a mental not to make sure TQVaultAE is not running when I delete characters from the game in future.

Thank you for the info. :)
